src/ape/managers/accounts.py

```diff
@@ -15,6 +15,7 @@
 
 class TestAccountManager(list, ManagerAccessMixin):
     __test__ = False
+    _num_generated = 0
```

Comment:
Can this be moved to `TestAccountContainer`?

Reply:
@fubuloubu this is the thing i mentioned in discord also. if i move this to TestAccountContainer, then this num_generated is not changing because containers is a property that is reinitializing all containers everytime you hit and meaning num_generated will be 0 all the time, so in tests, if i want to have 3 different accounts i cant have it.

Reply:
Good catch. As mentioned in discord, I believe this can be resolved by using `cached_property` for `.containers` property in `TestAccountManager`. I believe this to be an oversight from a previous feature implementation that is causing the issue you've encountered